lib/models/ActorSchema.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+
+const actorSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  dob: Date,
+  pob: String
+});
+
+actorSchema.statics.findByIdWithActors = function(id) {
```

Comment:
Is it .findByIdWithActors() of withFilms()? Still works, just a naming suggestion.